plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -533,10 +581,11 @@ public Optional<ConnectorViewDefinition> getView(ConnectorSession session, Schem
         });
     }
 
-    private Optional<View> getIcebergView(ConnectorSession session, SchemaTableName viewName)
+    private Optional<View> getIcebergView(ConnectorSession session, SchemaTableName viewName, boolean findLiveView)
```

Comment:
The new logic is little hard to understand than before. I don't think code readers can understand the meaning of "Live" objects easily.

Reply:
how about using same method and decide based on `getCached` parameter as done in https://github.com/trinodb/trino/pull/23867/commits/6a5a0815186edacf616114c50697ac20edc5dacb